src/braket/ocean_plugin/braket_sampler.py

```diff
@@ -51,11 +51,19 @@ class BraketSampler(Sampler, Structured):
 
     def __init__(
         self,
-        s3_destination_folder: AwsSession.S3DestinationFolder,
-        device_arn: str,
+        s3_destination_folder: AwsSession.S3DestinationFolder = None,
+        device_arn: str = None,
         aws_session: AwsSession = None,
         logger: Logger = getLogger(__name__),
     ):
+        if not device_arn:
+            try:
+                device_arn = AwsDevice.get_devices(
+                    provider_names=["D-Wave Systems"], statuses=["ONLINE"]
+                )[0].arn
+            except IndexError:
+                raise RuntimeError("No D-Wave devices online")
```

Comment:
Why are we making this change ?

Reply:
We want the `s3_destination_folder` parameter to be optional. To accomplish this, we either need to switch the order of parameters (a breaking change), or provide a default `device_arn` behavior (in parity with the BraketDwaveSampler class). After syncing with Christian, we determined the non-breaking approach was preferred